### PR TITLE
Add website links

### LINKED ISSUE #73: Website links
The Tabletop website has many interactive elements. For example, members can view the society's inventory as well as suggest board games to be added to it. Commands could be made to easily link to pages like these when relevant:

`.inventory` -> posts a link to https://www.warwicktabletop.co.uk/inventory/boardgames/
`.suggest boardgame` -> posts a link to https://www.warwicktabletop.co.uk/inventory/boardgames/suggestions/

### PR DESCRIPTION
**Pull request by joshdavies14**
Closes #73, adds `.inventory` and `.suggest boardgames` as basic commands

image

### CODE REVIEW

**Review by distributive — CHANGES_REQUESTED** [review]
As discussed, please create a new plugin for `suggest` and add `boardgame` as a subcommand (along with `ccg`, `wargame`, and `rpg`)

File: src/Tablebot/Plugins/Suggest.hs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          ["wargame"] -> return "Want to suggest a wargame? Here you go! -> https://www.warwicktabletop.co.uk/inventory/wargames/suggestions/"
+          ["rpg"] -> return "Want to suggest a RPG? Here you go! -> https://www.warwicktabletop.co.uk/inventory/rpgs/suggestions/"
+          _ -> return "Invalid input! Please try again with one of the following: `boardgame`, `ccg`, `wargame`, or `rpg`"
+        sendMessage m $ pack c
+
+suggestHelp :: HelpPage
+suggestHelp = HelpPage "suggest" [] "show links to suggest a new game for the society to buy" "**Suggest**\nShows the link to suggest a new game for the society to buy\n\n*Usage:* \n- `suggest boardgame` -> get the link to suggest a new boardgame to buy\n- `suggest ccg` -> get the link to suggest a new CCG to buy\n- `suggest wargame` -> get the link to suggest a new wargame to buy\n- `suggest rpg` -> get the link to request a new RPG to buy" [] None
+
+-- | @pingPlugin@ assembles these commands into a plugin containing both ping
+-- and pong.
```
Comment (finnbar) [inline src/Tablebot/Plugins/Suggest.hs:40]:
```suggestion

```
This comment is a bit wrong, and doesn't really add anything to the documentation by being there tbh

File: src/Tablebot/Plugins/Basic.hs [hunk 2]
```diff
@@ -58,6 +58,10 @@ basicCommands =
     ( "about",
       "This bot was created by finnbar to replace a couple of other bots in Tabletop. It's written in Haskell, and you can find the github here: <https://github.com/WarwickTabletop/tablebot>. There are setup guides and a contributor's guide to help you get started.",
       Simple ("some information about the bot", "Some information about the bot, including how you can get involved")
+    ),
+    ( "inventory",
+      "Here's the board games we possess! -> https://www.warwicktabletop.co.uk/inventory/boardgames/",
+      Simple ("the inventory", "The inventory, with a link to the actual inventory")
```
Comment (finnbar) [inline src/Tablebot/Plugins/Basic.hs:64]:
```suggestion
      Simple ("our board games inventory", "Our board games inventory, with a link to the actual inventory")
```
Tiny edit for clarification there - we technically have other inventories, but they're used a lot less.

**Review by finnbar — APPROVED** [review]
(puts on CS141 review hat) solid, clear code! I've added two suggestions (you can just click on them to commit them) to fix some of the text the commands output, but this is just me being precise with terminology

**Pull request merged**